### ISSUE TITLE
metrics calculated on the validation set or on the training set

### ISSUE THREAD

**Issue by inginfor** [ultralytics/yolov5#11959]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

but the metrics saved in the exp folder, in results.png i.e. metrics/precision, metrics/recall, metrics/mAP_0.5 and metrics/mAP_0.5:0.95 are calculated on the training set or on the validation set? 
<img width="488" alt="Screenshot 2023-08-09 182943" src="https://github.com/ultralytics/yolov5/assets/141730966/5fb241db-8d2b-4d18-a719-b35fb891d1e4">
<img width="631" alt="Screenshot 2023-08-09 183000" src="https://github.com/ultralytics/yolov5/assets/141730966/6b054e9a-024f-4ce7-bce9-ae9f08db3fd5">

**Comment by glenn-jocher** [comment]
@inginfor the metrics saved in the `metrics` folder, such as precision, recall, mAP_0.5, and mAP_0.5:0.95, are calculated on the validation set. This allows you to evaluate the performance of the model on data that it hasn't been trained on. If you have any further questions, feel free to ask.

**Comment by inginfor** [comment]
<img width="851" alt="image" src="https://github.com/ultralytics/yolov5/assets/141730966/6d6e4dca-112e-4ecb-8960-339821f3a8ab">
These metrics calculated during network training are calculated on the training set or on the validation set?

**Comment by glenn-jocher** [comment]
@inginfor The metrics calculated during network training, such as precision, recall, mAP_0.5, and mAP_0.5:0.95, are calculated on the training set. This helps to monitor the model's performance during the training process. If you have any other questions, please feel free to ask.
